sample/QueueTrigger-Powershell/function.json

```diff
@@ -1,17 +1,19 @@
-﻿{
-    "bindings": [
-        {
-            "type": "queueTrigger",
-            "direction": "in",
-            "queueName": "samples-powershell"
-        },
-        {
-            "type": "table",
-            "name": "output",
-            "direction": "out",
-            "tableName": "samples",
-            "partitionKey": "samples-powershell",
-            "rowKey": "{id}"
-        }
-    ]
-}
+﻿{
+  "bindings": [

```

Comment:
Though I think we did decide to standardize on 2 spaces instead of 4. Should not be part of this change, but we should do a cleanup pass on all of them.